src/sfctl/tests/request_generation_test.py

```diff
@@ -358,6 +358,37 @@ def paths_generation_helper(self):  # pylint: disable=too-many-statements
             '{"CreateFabricDump":"True", "NodeInstanceId":"ID"}',
             validate_flat_dictionary)
 
+        # container commands
+        self.validate_command( # get container logs
+            'sfctl container invoke-api --node-name Node01 --application-id samples/winnodejs '
+            '--service-manifest-name NodeServicePackage --code-package-name NodeService.Code '
+            '--code-package-instance-id 131668159770315380 --container-api-uri-path "/containers/{id}/logs?stdout=true&stderr=true"',
+            'POST',
+            '/Nodes/Node01/$/GetApplications/samples/winnodejs/$/GetCodePackages/$/ContainerApi',
+            ['api-version=6.2', 'ServiceManifestName=NodeServicePackage', 'CodePackageName=NodeService.Code', 'CodePackageInstanceId=131668159770315380', 'timeout=60'],
+            ('{"UriPath": "/containers/{id}/logs?stdout=true&stderr=true"}'),
+            validate_flat_dictionary)
+        self.validate_command( # get container logs
+            'sfctl container logs --node-name Node01 --application-id samples/winnodejs '
+            '--service-manifest-name NodeServicePackage --code-package-name NodeService.Code '
+            '--code-package-instance-id 131668159770315380',
+            'POST',
+            '/Nodes/Node01/$/GetApplications/samples/winnodejs/$/GetCodePackages/$/ContainerApi',
+            ['api-version=6.2', 'ServiceManifestName=NodeServicePackage', 'CodePackageName=NodeService.Code', 'CodePackageInstanceId=131668159770315380', 'timeout=60'],
+            ('{"UriPath": "/containers/{id}/logs?stdout=true&stderr=true"}'),
+            validate_flat_dictionary)
+        # self.validate_command( # update container
```

Comment:
This is still failing?

Reply:
Yes, it still fails

Reply:
It looks related to the json string for --container-api-body 

INFO:clicore.testsdk:Command "sfctl container invoke-api --node-name N0020 --application-id nodejs1 --service-manifest-name NodeOnSF --code-package-name Code --code-package-instance-id 131673596679688285 --container-api-uri-path "/containers/{id}/update" --container-api-http-verb=POST --container-api-body "{ "CpuShares": 512}"" => 0. Output:
DEBUG:vcr.cassette:Exiting context for cassette at paths_generation_test.json.
DEBUG:vcr.cassette:Exiting context for cassette at X:\git\service-fabric-cli\src\sfctl\tests\recordings\test_paths_generation.yaml.
ERROR

======================================================================
ERROR: test_paths_generation (sfctl.tests.request_generation_test.ServiceFabricRequestTests)
This test calls all commands that exist within sfctl.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "X:\git\service-fabric-cli\src\sfctl\tests\request_generation_test.py", line 180, in test_paths_generation
    self.paths_generation_helper()
  File "X:\git\service-fabric-cli\src\sfctl\tests\request_generation_test.py", line 390, in paths_generation_helper
    validate_flat_dictionary)
  File "x:\git\service-fabric-cli\env\lib\site-packages\mock\mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "X:\git\service-fabric-cli\src\sfctl\tests\request_generation_test.py", line 142, in validate_command
    expected_obj = json.loads(body)
  File "c:\users\leikong\appdata\local\programs\python\python36\Lib\json\__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "c:\users\leikong\appdata\local\programs\python\python36\Lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "c:\users\leikong\appdata\local\programs\python\python36\Lib\json\decoder.py", line 355, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting ',' delimiter: line 1 column 72 (char 71)